Utils/Dataflow/030_PDFAnalyzer/xmltable.py

```diff
@@ -247,9 +244,9 @@ def construct_row(self, line, used_lines):
         Lines which were already used are not used again.
         """
         row = [line]
-        for l in self.lines:
-            if l != line and l not in used_lines and line.same_row(l):
-                row.append(l)
+        for ln in self.lines:
```

Comment:
I'd rename "line" from the function parameters to "first_line" and use "line" as the replacement for "l" (or the new "ln"): makes it more obvious on what's the meaning of the parameter and avoids confusion with iteration variable.

Reply:
This sounds like a good idea to me, thanks.